homework_5/task001.py

```diff
@@ -0,0 +1,14 @@
+# Напишите программу, удаляющую из текста все слова, в которых присутствуют все буквы "абв".
+
+text = "Научная фантастика может быть полезной — она стимулирует воображение и избавляет от страха перед будущим."
+text = text.split()
+print(text)
+result_text = []
+for i in range(len(text)):
+    if "а" in text[i].lower() and "б" in text[i].lower() and "в" in text[i].lower():
+        continue
+    else:
+        result_text.append(text[i])
```

Comment:
```suggestion
for i in range(len(text)):
    if not ("а" in text[i].lower() and "б" in text[i].lower() and "в" in text[i].lower()):
        result_text.append(text[i])
```
Результат тот же, но на 2 строки меньше.